File: solutions/go/pangram/1/pangram.go
package pangram

import (
    "strings"
    "slices"
    "unicode"
)

func IsPangram(input string) bool {
    currents := make([]rune, 26)
    i := 0
    for _, r := range strings.ToLower(input) {
        if unicode.IsLetter(r) && slices.Contains(currents, r) == false {
            currents[i] = r
            i++
        }
    }

    if i < 26 {
        return false
    }
    
    return true    
}

File: solutions/csharp/pangram/1/Pangram.cs
public static class Pangram
{
    public static bool IsPangram(string input)
    {
        var letters = new HashSet<char>("abcdefghijklmnopqrstuvwxyz");
        var lowerInput = input.ToLower();
        foreach (var l in letters)
        {
            if (!lowerInput.Any(c => c == l))
                return false;
        }
        return true;
    }
}
